vllm/v1/request.py

```diff
@@ -93,9 +93,11 @@ def append_output_token_ids(
         token_ids: Union[int, list[int]],
     ) -> None:
         if isinstance(token_ids, int):
-            token_ids = [token_ids]
```

Comment:
Avoid creating temporary list (this method is currently only ever called with a single token)